packages/host/app/services/realm.ts

```diff
@@ -277,6 +277,10 @@ export default class RealmService extends Service {
     return this.knownRealm(url)?.isPublic ?? false;
   };
 
+  isIndexCard = (url: string): boolean => {
+    return url.endsWith('/index');
+  };
+
```

Comment:
We already have an `isIndex` card so let's try to not reimplement this twice. perhaps this resource can leverage the same method? https://github.com/cardstack/boxel/blob/main/packages/host/app/resources/auto-attached-card.ts#L79-L81.

Also, note the `isIndexCard()` in the auto-attached-card is more precise. specifically it is the `${realmURL}/index`. since you are operating in the realm service you know all the known realms. you should assert that the URL is one of the known realms followed by "/index"

Reply:
Thanks @habdelra. Following your suggestion, I extracted isIndexCard() as a helper, and it now resides in the stack item library file. Let me know if the latest changes look natural to you